Bottom: 036f5cba9ebc2b610ebfbb0db6586721d5c9a18d
Top:    4d692d318c19d5aeb0d533ba782e45c9cee3b18e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-12 01:31:03 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 26b20cb..c6f27e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
